### PR TITLE
Add get Create iModel Operation Details operation

### PR DESCRIPTION
**Pull request by arminasbrazenas**
- Added `Get Create iModel Operation Details` operation.
- Fixed Changeset Group integration tests. Reusable iModel's changeset groups were getting in state `timedOut` due to not being completed in time. To fix it, I added code which completes reusable iModel's changeset groups immediately after uploading the changesets.

### CODE REVIEW

File: tests/imodels-clients-tests/src/integration/management/OperationOperations.test.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      authorization,
+      iModelId: testIModel.id
+    };
+
+    // Act
+    const operationDetails = await iModelsClient.operations.getCreateIModelDetails(operationParams);
+
+    // Assert
+    expect(operationDetails.clonedFrom).to.be.null;
+    expect(operationDetails.state).to.be.oneOf([IModelCreationState.Scheduled, IModelCreationState.Successful]);
```
Comment (austeja-bentley) [inline tests/imodels-clients-tests/src/integration/management/OperationOperations.test.ts:43]: `testIModel` should be initialized by the time this test runs, so we should only expect successful state

File: utils/imodels-client-test-utils/src/test-context-providers/imodel/TestIModelCreator.ts [hunk 2]
```diff
@@ -58,8 +58,7 @@ export class TestIModelCreator {
   public async createEmptyAndUploadChangesets(iModelName: string): Promise<IModelMetadata> {
     const iModel = await this.createEmpty(iModelName);
     const briefcase = await this.acquireBriefcase(iModel.id);
-    const changesetGroups = await this.createChangesetGroups(iModel.id);
```
Comment (austeja-bentley) [inline utils/imodels-client-test-utils/src/test-context-providers/imodel/TestIModelCreator.ts:61]: why was this deleted?

File: clients/imodels-client-management/src/base/types/apiEntities/OperationInterfaces.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  Failed = "failed"
+}
+
+/** Information about the source iModel of an iModel clone. */
+export interface ClonedFrom {
+  /** Id of the source iModel. */
+  iModelId: string;
+  /**
+   * Id of the latest source iModel Changeset which was copied to this iModel.
+   * This corresponds to the Changeset specified in changesetId or changesetIndex properties when cloning an iModel.
```
Comment (austeja-bentley) [inline clients/imodels-client-management/src/base/types/apiEntities/OperationInterfaces.ts:24]: I think we surround property names with `` in jsdoc comments, at least most of the time

File: utils/imodels-client-test-utils/src/test-context-providers/imodel/TestIModelCreator.ts (hunk 2 above)
Comment (arminasbrazenas) [reply utils/imodels-client-test-utils/src/test-context-providers/imodel/TestIModelCreator.ts:61]: I added it with changeset groups, but it's not required for any tests that use the created iModel. Therefore, it should save a few (useless) calls to API for integration tests. In my opinion, we can add it when we need it.

File: clients/imodels-client-management/src/base/types/apiEntities/OperationInterfaces.ts (hunk 3 above)
Comment (arminasbrazenas) [reply clients/imodels-client-management/src/base/types/apiEntities/OperationInterfaces.ts]: updated

File: tests/imodels-clients-tests/src/integration/management/OperationOperations.test.ts (hunk 1 above)
Comment (arminasbrazenas) [reply tests/imodels-clients-tests/src/integration/management/OperationOperations.test.ts]: fixed

**Review by austeja-bentley — APPROVED** [review]
